roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_predicates.py

```diff
@@ -98,7 +98,7 @@ def run(self, terms, variables=None, regions_enabled=True, short_version=None,
                 {'name': 'MatchInterPodAffinity'}
             ])
 
-        if short_version in ['1.5', '1.6']:
+        if short_version in ['1.5', '1.6', 'latest']:
```

Comment:
Instead of doing it this way, should we instead hardcode latest to a specific version in these lookup plugins? We already need to audit these for each release, so it seems like having to update the version associated with latest shouldn't be much more of a burden.